emulators/qemu/Portfile

```diff
@@ -25,15 +25,17 @@ homepage                https://www.qemu.org
 master_sites            https://download.qemu.org/
 use_xz                  yes
 
-checksums               rmd160  58a704960cd712cf6c218216e8426a05948f6526 \
-                        sha256  2f13a92a0fa5c8b69ff0796b59b86b080bbb92ebad5d301a7724dd06b5e78cb6 \
-                        size    62426192
+checksums               rmd160  2c33e773f012e333f99237e3d4ff1653ea0bc88f \
+                        sha256  cb18d889b628fbe637672b0326789d9b0e3b8027e0445b936537c78549df17bc \
+                        size    106902800
 
 patchfiles              patch-configure.diff
 patch.pre_args          -p1
 
-depends_build           port:texinfo \
+depends_build           port:perl5 \
```

Comment:
As perl is not linked, I would prefer to move it to the python section with `license_noconflict` below.